<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureShow</class>
 <widget class="QMainWindow" name="PictureShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureShow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="PicPreConfig" name="id_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="PicPreview" name="id_frame">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="id_move_frame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PicPreview</class>
   <extends>QFrame</extends>
   <header location="global">picpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PicPreConfig</class>
   <extends>QWidget</extends>
   <header>picpreconfig.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
